client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -121,14 +136,19 @@ export default React.createClass( {
 			return ( <div className="plugin-meta__actions"> { this.getInstallButton() } </div> );
 		}
 
+		const {
+			autoupdate: canToggleAutoupdate,
+			activation: canToggleActivation,
+			remove: canRemove,
+		} = this.props.allowedActions;
 		return (
 			<div className="plugin-meta__actions">
-				<PluginActivateToggle plugin={ this.props.plugin } site={ this.props.selectedSite }
-					notices={ this.props.notices } isMock={ this.props.isMock } />
-				<PluginAutoupdateToggle plugin={ this.props.plugin } site={ this.props.selectedSite }
-					notices={ this.props.notices } wporg={ this.props.plugin.wporg } isMock={ this.props.isMock } />
-				<PluginRemoveButton plugin={ this.props.plugin } site={ this.props.selectedSite }
-					notices={ this.props.notices } isMock={ this.props.isMock } />
+				{ canToggleActivation && <PluginActivateToggle plugin={ this.props.plugin } site={ this.props.selectedSite }
```

Comment:
These lines are increasing too much. Could we separate the properties a little to improve reading? Something like ...

```es6
{ canToggleActivation && <PluginActivateToggle
  plugin={ this.props.plugin }
  site={ this.props.selectedSite }
  notices={ this.props.notices }
  isMock={ this.props.isMock } />
}
```

Not a big deal. Feel free to ignore this comment.